src/forms/button/Button.tsx

```diff
@@ -70,14 +70,14 @@ export class Button extends React.PureComponent<ButtonProps> {
     }
 
     render() {
-        const {disabled, children, onClick, onSubmit, submit, icon, show} = this.props;
+        const {disabled, children, onClick, onSubmit, icon, show, type} = this.props;
         return show ? (
             <button
                 disabled={disabled}
                 className={classNames(Button.getClassNames(this.props))}
                 onClick={onClick}
                 onSubmit={onSubmit}
-                type={submit ? "submit" : undefined}
+                type={type ? type : undefined}
```

Comment:
empty string is not valid? I mean if type is empty then undefined will be passed

Reply:
done